lib/ansible/plugins/filter/password_hash.yml

```
@@ -7,6 +7,7 @@ DOCUMENTATION:
   positional: _input
   notes:
     - Algorithms available might be restricted by the system.
+    - Algorithms may restrict salt length or content. For example, bcrypt/blowfish requires a 22-character salt.
```

If these are values, they should use `V(...)`:
```suggestion
    - Algorithms may restrict salt length or content. For example, V(bcrypt)/V(blowfish) requires a 22-character salt.
```
Otherwise the algorithms should be properly capitalized:
```suggestion
    - Algorithms may restrict salt length or content. For example, bcrypt/Blowfish requires a 22-character salt.
```